Ticket - 29137 Sharp shooter Label Menu FG - print is not working

sewa67 opened the pull request:
Please review this change and merge into development branch 
Files - addon/loadtags/d-tagprt.w

sewa67 committed:
Ticket - 29137 Sharp shooter Label Menu FG - print is not working
Files - addon/loadtags/d-tagprt.w

brad-advantzware reviewed (changes requested):
This is copied code.  I would like to just run the standard loadtag printing.  Comments in ticket.

sewa67 committed:
Ticket - 29137 Sharp shooter Label Menu FG - print is not working
Files -  oe/rep/lodxprntstd.i,oe/rep/lodxprnt.i,oe/rep/lodxprnt2.i,oe/rep/lodxprnt3.i,oe/rep/lodxprnt4.i,oe/rep/lodxprnt5.i,
               oe/rep/lodxprnt6.i,oe/rep/lodxprnt7.i,oe/rep/lodxprnt8.i,oe/rep/lodxprnt9.i,oe/rep/lodxprnt10.i,oe/rep/lodxprnt11.i,
              oe/rep/lodxprnt12.i,oe/rep/lodxprnt13.i,oerep/LoadtagProcs.p,oerep/r-loadtg.w,addon/loadtags/d-tagprt.w
